ui/src/NotificationContext.tsx

```diff
@@ -28,7 +28,7 @@ const NotificationContext = createContext<INotificationContext>({
   },
 });
 
-export const NotificationProvider: FC<{}> = ({ children }) => {
+export const NotificationProvider: FC = ({ children }) => {
```

Comment:
I just reviewed https://github.com/docker/mercury-ui/pull/3852 which advocates to use function declaration over function expression. I guess it is too late to change it everywhere, but should we consider to move everything to the function declaration?

Reply:
Yes, I believe this could have a bigger impact on large codebases (pinata / hub) and although by having it here we can have consistency on how we write our code I wouldn't worry too much about extensions (or at least the ones already built). When we have a linter rule that forces that we can include it here as well. WDYT?